src/test/java/org/betonquest/betonquest/modules/logger/custom/debug/config/SimpleDebugConfigTest.java

```diff
@@ -0,0 +1,98 @@
+package org.betonquest.betonquest.modules.logger.custom.debug.config;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Test for a {@link SimpleDebugConfig}.
+ */
+class SimpleDebugConfigTest {
+    /**
+     * Assertion message when order is not correct.
+     */
+    public static final String EXPECTED_COUNT_MESSAGE = "Expected correct order checked by this counter";
+
+    /**
+     * Default constructor.
+     */
+    public SimpleDebugConfigTest() {
+        // Empty
+    }
+
+    @Test
+    @SuppressWarnings("PMD.JUnitTestContainsTooManyAsserts")
+    void testEnableDisable() throws IOException {
+        final DebugConfig debugConfig = new SimpleDebugConfig(false);
+        assertTestEnableDisable(debugConfig);
```

Comment:
The called method is not an assertment but a test. Split the test into units and when grouping assertments into a function then that function should only contain assertments (and sometimes a loop).